backend/model/login.php

```diff
@@ -9,7 +9,7 @@
 define('ALGORITHM','HS512');
 
 class Login{
-
+  private $emailRegex = '/^[a-zA-Z]+[0-9]*@mtech\\.edu$/';
   private $domain = 'katie.mtech.edu/~acmuser';
```

Comment:
Currently have configs for backend as a phase two item. Will need to look through the frontend configs and do some thinking for that.